pandas/tests/extension/arrow/test_bool.py

```diff
@@ -50,6 +50,10 @@ def test_view(self, data):
         # __setitem__ does not work, so we only have a smoke-test
         data.view()
 
+    @pytest.mark.xfail(raises=AssertionError, reason="Not implemented yet")
+    def test_contains(self, data, data_missing):
+        super().test_contains(data, data_missing)
```

Comment:
The Arrow arrays have Arrow Nulls as the dtype.na_value, while data_missing[0] gives ´None´.

Maybe @TomAugspurger could look into that as part of his Arrow work?

Reply:
> The Arrow arrays have Arrow Nulls as the dtype.na_value, while data_missing[0] gives ´None´.

It's fine to simply skip it as you did. This are only test EAs, that were needed for certain specific tests, so no problem this don't fully work otherwise. 

The actual public arrays using Arrow under the hood (eg the new ArrowStringArray) has a different implementation and is fully tested.

Reply:
Should we also base these test arrays on `pd.NA`?  Otherwise we can use the same code as in the string array to replace the scalar return value of `None` with `na_value`.